### DISCUSSION TITLE
What to do with '2 Trackers detected'?

### DISCUSSION THREAD

**Discussion by donni-h** [seemoo-lab/AirGuard#142] [Q&A]
Hey you guys,
I live in a multi-story house and it keeps telling me that 2 trackers are being detected (even though many more are within scan range).
Most of the tracked locations are close to my house, so the odds are pretty high that the Tag belongs to a neighbor of mine, and if it happens to be a bit further it is still not unreasonable to assume that I might cross path with said neighbor now and again.
For AirTag devices I could just play the sound to check, whether or not it is hidden in my apartment (even though I might startle my neighbors if it randomly goes off without warning). But how can I check TIle or similar devices? There is no option to play any sounds, and while the map of tracked locations is neat - I have no way to create a timeline out of them, as there seems to be no log available that has timestamps associated to the locations. So how would I proceed in such a scenario to make sure that no malicious device is in my proximity?

In any case, thank you for this awesome app and for making the world a bit better. :)

**Comment by Sn0wfreezeDev** [comment]
Hi,

Tile devices are prone to cause such issues, because they never change their identifier.
So if you see your neighbour every 2-3 days you will get that message at some point.
The good thing is that you can ignore them and we are also working on an enhanced scan that allows you to look for hidden scanners.
Playing a sound is currently not an option for Tile.

Kind regards,
Alexander Heinrich
